superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -290,7 +290,7 @@ export default class FilterableTable extends PureComponent<
   }
 
   fitTableToWidthIfNeeded() {
-    const containerWidth = this.container.current!.clientWidth;
+    const containerWidth = this.container?.current?.clientWidth ?? 0;
```

Comment:
@etr2460 check my math on this one, but I think this should work to the same effect.

Reply:
it might be better to short circuit the whole function if the container ref isn't set up?

Reply:
Addressed... let me know if you had a different approach in mind. Thanks :)

Reply:
Just kidding... what I did caused another TS error that my IDE didn't catch.

Reply:
The cascade of TS workarounds turned out to be nearly as goofy... so I just pulled the function's logic into `componentDidMount` since it's not used elsewhere anyway... and all is well.

Reply:
j/k, that doesn't work either... back to silly workarounds. Sorry for all the github/email noise. Let's both hope this PR goes away soon.